cmd/auth/auth.go

```diff
@@ -184,67 +94,65 @@ func verifyAPIKey(datarobotHost string, apiKey string) (bool, error) {
 	return resp.StatusCode == http.StatusOK, nil
 }
 
-func LoginAction() error { //nolint: cyclop
+func LoginAction() error {
 	reader := bufio.NewReader(os.Stdin)
 
-	if err := createConfigFileDirIfNotExists(); err != nil {
+	err := config.ReadConfigFile("")
+	if err != nil {
 		panic(err)
 	}
 
-	fileInfo, _ := os.Stat(configFilePath)
-
 	datarobotHost, err := GetURL(false)
 	if err != nil {
 		panic(err)
 	}
 
-	if fileInfo.Size() > 0 { //nolint: nestif
-		currentKey, err := GetAPIKey()
-		if err != nil {
-			panic(err)
-		}
+	currentKey := viper.GetString(DataRobotAPIKey)
+	if currentKey == "" {
+		panic("API key is empty, please log in to DataRobot first.")
+	}
 
-		isValidKeyPair, err := verifyAPIKey(datarobotHost, currentKey)
+	isValidKeyPair, err := verifyAPIKey(datarobotHost, currentKey)
+	if err != nil {
+		panic(err)
+	}
+
+	if isValidKeyPair {
+		fmt.Println("An API key is already present, do you want to overwrite? (y/N): ")
+
+		selectedOption, err := reader.ReadString('\n')
 		if err != nil {
 			panic(err)
 		}
 
-		if isValidKeyPair {
-			fmt.Println("An API key is already present, do you want to overwrite? (y/N): ")
-
-			selectedOption, err := reader.ReadString('\n')
-			if err != nil {
-				panic(err)
-			}
-
-			if strings.ToLower(strings.Replace(selectedOption, "\n", "", -1)) == "y" {
-				// Set the DataRobot API key to be an empty string
-				if err := setValueInConfigFile(DataRobotAPIKey, ""); err != nil {
-					panic(err)
-				}
-			} else {
-				fmt.Println("Exiting without overwriting the API key.")
-				return nil
-			}
+		if strings.ToLower(strings.Replace(selectedOption, "\n", "", -1)) == "y" {
+			// Set the DataRobot API key to be an empty string
```

Comment:
Is this comment correct?
Sorry maybe I'm wrong but it looks like `currentKey` is _not_ an empty string at this point?

Reply:
Sorry this comment still says we're setting it to an empty string but is `currentKey` actually an empty string here?